aos/closed_loop/ClosedLoopSimulation.ipynb

```diff
@@ -9,19 +9,18 @@
     "\n",
```

Comment:
<p>Is there a reason you don't want to include all the plots from before? I really like the illustration of the Wavefront Errors getting corrected after the iteration of the closed loop.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_analysis_notebooks/pull/25/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>